extrinsic_matrix:
  - [-0.707,  0.683, -0.183, -0.569]
  - [ 0.035, -0.224, -0.974, -0.452]
  - [-0.706, -0.695,  0.135,  0.036]
  - [ 0.000,  0.000,  0.000,  1.000]